feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -446,48 +438,65 @@ fun QuickSettingsGrid(
  * The top bar indicators for quick settings items.
  */
 @Composable
-fun Indicator(enum: QuickSettingsEnum, onClick: () -> Unit, modifier: Modifier = Modifier) {
-    Icon(
-        painter = enum.getPainter(),
-        contentDescription = stringResource(id = enum.getDescriptionResId()),
-        modifier = modifier
-            .size(dimensionResource(id = R.dimen.quick_settings_indicator_size))
-            .clickable { onClick() }
-    )
+fun Indicator(
```

Comment:
nit: could we just rename Indicator to something like `TopBarSettingUI` or `PreviewSettingUI`?  The name `Indicator` used as a template like this doesn't feel right

Reply:
Done.